Technical-Steering-Committee/proposal.md

```diff
@@ -0,0 +1,89 @@
+# Zowe Technical Steering Committee Proposal
+
+ - [Background](#background)
+ - [Role](#role)
+ - [Composition](#composition)
+ - [Operations and Processes](#operations-and-processes)
+   - [Chairperson and Secretary](#chairperson-and-secretary)
+   - [Meetings](#meetings)
+   - [Voting](#voting)
+
+The Zowe project is hosted by the Open Mainframe Project and part of the Linux Foundation which operates transparently, openly, collaboratively and ethically.  
+
+This proposal is to form a Zowe Technical Steering Committee (TSC) to oversee, govern, and improve the infrastructure, execution, and customer experience related to development, testing, packaging, delivery, and cross-squad technology overlap areas.  
+
+## Background
+
+When Zowe was first created different technology from different codebase combined and was stewarded by individual squads who oversaw each component.  Each squad was responsible for tracking and planning the backlog of items for their component.
+
+As Zowe has grown, coordination between squads has grown, resulting in a common installer for z/OS components, and more recently features such as Sign Sign On (SSO), MultiFactor Authentication (MFA) and client-side certificates have required technical and delivery coordination between the different components. These capabilities have enhanced Zowe’s value to the community.  It is anticipated this vector of integration will continue in the future with high availability (HA), containerization, improved security and other cross component features.  
+
+The coordination and synchronization of the moving parts of Zowe is becoming more central to successful delivery of features and for this reason the Zowe Leadership Committee (ZLC) taking leadership guidance from other successful Linux Foundation projects is proposing the creation a Technical Steering Committee (TSC).
+
+## Role
+
+The TSC will oversee the technical direction of the Zowe project. Responsibilities include, but are not limited to
+
+- Setting the release schedule, release quality standards, and approving releases.
```

Comment:
This pulled together a few release points from before. I did remove the bit about incrementing the release number, as it seemed a bit odd that the ZLC would be involved with that.

Reply:
The reason the ZLC is involved with the release number increment, is because the ZLC has the business perspective on the impact that makes to customers and ISVs with conformant plugins.  A new version number of Zowe mandates a new SMP/E installation that is new FMID that causes customer disruption as it is effectively a new isolated Zowe unrelated to the previous version's installation, and it also will cause a new LTS program with ramifications for supporting the old version and conformant plugins for the old version will need to re-apply for conformance to the new version and also adapt to any breaking changes.  
For this reason moving to a new version number of Zowe is something that must be taken with all of the input from relevent parties which is why it's included in the proposal as being a ZLC decision taking recommendation from the TSC.

Reply:
Who are the relevant parties that are involved in the decision to increment the version? And is this just for major versions, or minor/patch versions as well?